Gifski/GIFGenerator.swift

```diff
@@ -380,6 +380,92 @@ extension GIFGenerator.Conversion {
 			return .seconds(bounce ? (duration * 2) : duration)
 		}
 	}
+
+	var videoWithoutBounceDuration: Duration {
+		get async throws {
+			.seconds(try await gifDuration.toTimeInterval / (bounce ? 2 : 1))
+		}
+	}
+
+	/**
+	- Returns: The current scale of the `dimensions` compared to the dimensions of the video track.
+	 */
+	var scale: CGSize {
+		get async throws {
+			guard let trackSize = try await trackSize else {
+				return .one
+			}
+			guard let dimensions = dimensionsAsSize else {
+				return trackSize
+			}
+			return dimensions / trackSize
+		}
+	}
+
+	/**
+	- Returns: Dimensions of the first video track
+	 */
```

Comment:
```suggestion
	*/
```

same in many places.

Look over the diff for nitpick like this: https://github.com/sindresorhus/Gifski/pull/339/files